packages/theme/src/ThemeProvider.tsx

```diff
@@ -7,9 +7,12 @@ import type { DefaultTheme } from 'styled-components';
 import defaultTheme from './theme';
 
 const containerStyles = {
-  display: 'grid',
-  minHeight: '100svh',
-  gridTemplateRows: '1fr'
+  backgroundColor: '#000000',
+  color: defaultTheme.allColors.plainBg,
```

Comment:
I'm not sure, but i think that the theme should be smart enough to detect these values w/out setting them here. isn't there a `bg` and `fg` value in the theme?